backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Docs_Shipping_Notification_Description.sql

```diff
@@ -0,0 +1,83 @@
+/*
+ * #%L
+ * de.metas.fresh.base
+ * %%
+ * Copyright (C) 2023 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+DROP FUNCTION IF EXISTS de_metas_endcustomer_fresh_reports.Docs_Shipping_Notification_Description(record_id   numeric,
+                                                                                                  ad_language character varying)
+;
+
+CREATE FUNCTION de_metas_endcustomer_fresh_reports.Docs_Shipping_Notification_Description(record_id   numeric,
+                                                                                          ad_language character varying)
+    RETURNS TABLE
+            (
+                DeliveryTo            character varying,
+                Document_name         character varying,
+                documentno            character varying,
+                sales_order           character varying,
+                poreference           character varying,
+                physicalclearancedate timestamp WITHOUT TIME ZONE,
+                Harvesting_year       character varying,
+                description           character varying
+            )
+    STABLE
+    LANGUAGE sql
+AS
+$$
+SELECT
+       TRIM(COALESCE(snbp.name || E'\n', '') ||
+            COALESCE(snbploc.address1 || E'\n', '') ||
+            COALESCE(snbploc.postal || E'\n', '') ||
+            COALESCE(snbploc.city || E'\n', '') ||
+            COALESCE(snbpc.name, '')
+           )           AS DeliveryTo,
```

Comment:
Why creating this?
We have "de/metas/docs/generics/report_bp_org_block_left.jasper" for this exact purpose and is already used in report.jrxml

Reply:
`DeliveryTo` removed from description